### PR TITLE
Issue #52 Add global variables

### PR DESCRIPTION
**Pull request by OfirSiboni**
Hey,
added 2 global variables, `PRINT_FORMULA` and `PRINT_LATEX` so users can change them and not pass it every time they call the function.

### CODE REVIEW

File: pytexit/pytexit.py [hunk 1]
```diff
@@ -151,6 +152,12 @@ def py2tex(
 
     """
 
+    # Check print globals
+    if print_formula is None:
+        print_formula = g_print_formula
```
Comment (erwanp) [inline pytexit/pytexit.py:157]: Shouldn't this be the global variable?

Comment (OfirSiboni) [reply pytexit/pytexit.py:157]: oh, syntax error, fixing.

**Comment by erwanp** [comment]
Does your code work if you use
```
import pytexit
pytexit.PRINT_FORMULA = False

py2tex("...")
```
?

If so, you should mention such an example in the py2tex() docstring; and maybe in the Notes of the documentation ([edit this file](https://github.com/erwanp/pytexit/blob/master/docs/userguide.rst#notes) )  so that other users can easily use your changes

**Review by erwanp — APPROVED** [review]

**Comment by erwanp** [comment]
(linting error only, mergin)

**Pull request merged**